### ISSUE TITLE
Support out-of-order time-series events for Activity Aggregate

### ISSUE THREAD

**Issue by indradeep** [productml/blurr#215]
The Activity Aggregate currently expects events to arrive in order.  If events arrive out of order, it currently breaks the chain of sessions created.

Once the store normalization #187 is complete, the incorporate the new store APIs to build the out-of-order support.

**Comment by prashastk** [comment]
Duplicate of #179
